README.md

```diff
@@ -11,20 +11,77 @@ git clone https://github.com/piccolo-orm/piccolo-admin-docker.git
 Creating an `.env` file.
 
 ```bash
-cd app
 cp .env.example .env && rm .env.example
-cd ..
 ```
-Creating a Docker image.
+
+Run the Docker container.
 
 ```bash
-docker build -t piccolo_admin .
+docker-compose up -d
 ```
 
-Running a Docker image (use the `--network=host` flag for an existing database from the local machine).
+After site is running log in as admin user on [localhost:8000](http://localhost:8000) and use legacy database.
+
+Stop the Docker container.
 
 ```bash
-docker run -d --network=host --name admin_container piccolo_admin
+docker-compose down
 ```
 
-After site is running log in as admin user on [localhost:8000](http://localhost:8000/admin/) and use legacy database.
\ No newline at end of file
+### Additional Piccolo Admin configuration
+
+Piccolo Admin has a flexible UI with lots of configuration options to display only the columns you want your users to see. More information on Piccolo Admin [docs](https://piccolo-admin.readthedocs.io/en/latest/index.html).
+
+After Piccolo Admin is started with all the tables from the existing database, we can do additional configuration through the `config.yaml` file. 
+
+Example of `config.yaml`:
+
+```yaml
+tables:
+  # An example of additional Piccolo Admin configuration
+  Actor:
+    visible_columns:
```

Comment:
Maybe tweak this, so instead tables can either just be a string (the name of the table), or an object. For example:

``` yaml
-  city
-  name: address
    visible_columns:
      - address_id
      - address
      - city_id
    visible_filters:
      - address_id
      - address
      - city_id
    menu_group: Location
    rich_text_columns: address
```

So a user doesn't have to specify all of the options if they don't want to.